taxi/views.py

```diff
@@ -1,3 +1,16 @@
+from django.http import HttpResponse, HttpRequest
 from django.shortcuts import render
+from taxi.models import Driver, Manufacturer, Car
 
-# Create your views here.
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
+
+    context = {
+        "num_drivers": num_drivers,
+        "num_manufacturers": num_manufacturers,
+        "num_cars": num_cars
+    }
+    return HttpResponse(render(request, "taxi/index.html", context=context))
```

Comment:
The `render` function already returns an `HttpResponse` object, so you don't need to wrap it with `HttpResponse()`. You can directly return the result of `render()`.